frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -65,9 +68,61 @@ export const AccountsSummary = ({
   const [accountsTotalBalance, setAccountsTotalBalance] = useState<accountApi.TAccountsTotalBalance>();
   const [coinsTotalBalance, setCoinsTotalBalance] = useState<accountApi.TCoinsTotalBalance>();
   const [balances, setBalances] = useState<Balances>();
+  const { lightningConfig } = useLightning();
 
   const hasCard = useSDCard(devices);
 
+
+  const [lightningBalance, setLightningBalance] = useState<accountApi.IBalance>();
+  const fetchLightningBalance = useCallback(async () => {
+    setLightningBalance(await getLightningBalance());
+  }, []);
+
+  // if there is an active lightning account derived from this keystore, subscribe to any node state changes.
+  useEffect(() => {
+    const lightningAccounts = lightningConfig.accounts;
+    if (lightningAccounts.length) {
+      fetchLightningBalance();
+      const subscriptions = [subscribeNodeState(fetchLightningBalance)];
```

Comment:
When there is a node state change you should also fetch the coinsTotalBalance again, otherwise you miss the update there